conf/ConfigureFromEnv.php

```diff
@@ -139,7 +142,12 @@
 }
 
 if(defined('SS_ERROR_LOG')) {
-	SS_Log::add_writer(new SS_LogFileWriter(BASE_PATH . '/' . SS_ERROR_LOG), SS_Log::WARN, '<=');
+	$logger = Injector::inst()->get('Logger');
+	if($logger instanceof Logger) {
+		$logger->pushHandler(new StreamHandler(BASE_PATH . '/' . SS_ERROR_LOG, Logger::WARN));
```

Comment:
I disagree.

 * Less Code > More Code.
 * ConfigureFromEnv is currently the thing responsible for mapping env vars to internal config
 * The old implementation did this way; if we want to refactor ConfigureFromEnv, that's beyond the scope of this PR.